javascript/javascript1/week2/homework.js

```diff
@@ -1,15 +1,53 @@
 // Flight booking fullname function
-
 const getFullname = (firstname, surname, useFormalName) => {
   if (useFormalName) {
     return "Lord " + firstname + " " + surname;
   } else {
     return firstname + " " + surname;
   }
```

Comment:
Since you're catching on to Javascript so fast I'd like to challenge you to attempt to do this using the ternary operator & use template literals!

It will help adhere to the Javascript principal of DRY (Don't Repeat Yourself) as you wouldn't need to write `"firstName + " " + surname` twice.